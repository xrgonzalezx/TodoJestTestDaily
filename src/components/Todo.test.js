import React from 'react';
import renderer from 'react-test-renderer'
import Todo from './Todo'

it('renders without crashing', () => {
  const div = renderer.create(<Todo todo={{text: "test", id: 1, done: false}}/>).toJSON();
  expect(div).toMatchSnapshot();
});
