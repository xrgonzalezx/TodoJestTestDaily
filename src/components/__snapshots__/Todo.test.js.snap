// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Todo"
>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
  <span
    className="todo-not-done"
    onClick={[Function]}
  >
    test
  </span>
</div>
`;
